src/login/translation/messageIndex.js

```diff
@@ -46,15 +46,20 @@ export const unformattedMessages = defineMessages({
     defaultMessage: "Code is formatted as five groups of characters and numbers, separated by hyphens",
     description: "Help text for email confirmation code",
   },
-  "emails.invalid_code": {
-    id: "emails.invalid_code",
+  "modal.long_code_invalid_format": {
```

Comment:
perhaps "modal" could be removed from all ids as we already know that we want to remove the input from modals. When we do, we would have make changes here as well, so perhaps something more generic relating to "confirmation code" could be suitable?